samples/wifi/shell/README.rst

```diff
@@ -192,6 +205,29 @@ Supported CLI commands
        | wifi mode -g -i1
        | Set operation example for interface index 1 - set station+promiscuous
        | wifi mode -i1 -sp
+   * - packet_filter
+     - | This command is used to set packet filter setting when
+       | monitor, TX-Injection and promiscuous mode is enabled
+       | The different packet filter modes are control,
+       | management, data and enable all filters
```

Comment:
TX injection mode is edited as per https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/wifi/raw_tx_operation.html#tx-injection-mode for consistency.
```suggestion
   * - packet_filter
     - | This command is used to set packet filter settings when
       | monitor, TX injection, and promiscuous mode are enabled.
       | The different packet filter modes are control,
       | management, data, and enable all filters.
```